treeherder/config/settings.py

```diff
@@ -461,6 +461,11 @@
 
 # Only generate alerts for data newer than this time in seconds in perfherder
 PERFHERDER_ALERTS_MAX_AGE = timedelta(weeks=2)
+# From the same job's log, ingest (or not) multiple PERFHERDER_DATA dumps
+# pertaining to the same performance signature
+PERFHERDER_ENABLE_MULTIDATA_INGESTION = env.bool(
+    'PERFHERDER_ENABLE_MULTIDATA_INGESTION', default=True
```

Comment:
We might want to give this a default value of False, so that when its merged into master we can explicitly set this value in the heroku env settings once we're confident the schema changes have been successful.

Reply:
Ideally, the default behavior has this enabled, even if the env var is missing.
Either way, this var needs to be provided.

@sarah-clements could you make sure it is set on Heroku like so: `PERFHERDER_ENABLE_MULTIDATA_INGESTION=False` ?

Reply:
Ok, I've added it to all deployments.